cmd/bin/bin.go

```diff
@@ -0,0 +1,64 @@
+package main
```

Comment:
same as above, i would consider "faBin" or "binGene" as the filename if possible. Otherwise we'll have this weird construction of ~/go/bin/bin